client/src/actions/team.js

```diff
@@ -287,11 +278,11 @@ function teamDataAllocator(teamData) {
  */
 export const updateTeam = (teamId, teamData) => async (dispatch) => {
   try {
-    const { data } = await api.updateTeam(teamId, teamData);
-    const updatedData = teamDataAllocator(data);
+    const successMessage = await api.updateTeam(teamId, teamData)
+    dispatch(successMessageCreator(successMessage));
```

Comment:
Related to the above comment. We're dispatching an action with a payload that is set by the argument to this AC (i.e. `payload: {...teamData, teamId},`).